go.mod

```diff
@@ -498,3 +499,5 @@ replace github.com/googleapis/gnostic v0.5.6 => github.com/googleapis/gnostic v0
 
 // required to drop dependency on deprecated git.apache.org/thrift.git
 exclude go.opencensus.io v0.19.1
+
+replace github.com/signalfx/splunk-otel-collector/processor/timestampprocessor => ./pkg/processor/timestamp
```

Comment:
nit, do you mind sharing with the current tests replace directive to start a central location for the project components?